src/browser/core_state.js

```diff
@@ -512,6 +514,30 @@ function getAppAncestor(descendantAppUuid) {
     }
 }
 
+function setLicenseKey(identity, licenseKey) {
+    const { uuid } = identity;
+    const app = getAppByUuid(uuid);
+
+    if (app) {
+        app.licenseKey = licenseKey;
+
+        return licenseKey;
+    } else {
+        return null;
```

Comment:
Just a deterministic return as opposed to `undefined`  This also gives you an easy way to know if you actually set it w/o having to `throw`